temporalcli/commands.gen.go

```diff
@@ -473,10 +475,35 @@ func NewTemporalActivityFailCommand(cctx *CommandContext, parent *TemporalActivi
 	return &s
 }
 
+type ActivityReferenceOptions struct {
+	ActivityId   string
+	ActivityType string
+}
+
+func (v *ActivityReferenceOptions) buildFlags(_ *CommandContext, f *pflag.FlagSet, activityType string, onlyWithoutQuery bool) {
+	f.StringVarP(&v.ActivityId, "activity-id", "a", "", fmt.Sprintf("Activity ID to %s.", activityType))
+	if onlyWithoutQuery {
+		f.StringVarP(&v.ActivityType, "activity-type", "g", "", fmt.Sprintf("Activity Type to %s. Can only be used without --query.", activityType))
+	} else {
+		f.StringVarP(&v.ActivityType, "activity-type", "g", "", fmt.Sprintf("Activity Type to %s.", activityType))
+	}
+}
+
+func (v *ActivityReferenceOptions) validateFlags() error {
```

Comment:
You can't add code to this code-generated file, it'd get removed on regen. Rather, add in another file (or maybe better, do not add methods to generated structures beyond the run method needed on command, but make simple utility functions that accept them).

Reply:
Removed the `ActivityReferenceOptions` struct entirely, opting for the repetitive checking. The flags for this `activity-id` and `activity-type` are already set with other option-sets which causes a panic and unraveling that doesn't seem like the correct solution here.  

cc @ychebotarev